src/panels/config/logs/error-log-card.ts

```diff
@@ -16,52 +16,27 @@ import "../../../components/ha-ansi-to-html";
 import "../../../components/ha-card";
 import "../../../components/ha-icon-button";
 import "../../../components/ha-select";
-import { fetchErrorLog, LogProvider } from "../../../data/error_log";
+import { fetchErrorLog } from "../../../data/error_log";
 import { extractApiErrorMessage } from "../../../data/hassio/common";
 import { fetchHassioLogs } from "../../../data/hassio/supervisor";
 import { HomeAssistant } from "../../../types";
 
-const logProviders: LogProvider[] = [
-  {
-    key: "supervisor",
-    name: "Supervisor",
-  },
-  {
-    key: "core",
-    name: "Home Assistant Core",
```

Comment:
Will this fit on mobile?

Reply:
![CleanShot 2022-04-26 at 23 26 58](https://user-images.githubusercontent.com/18730868/165440705-2291c1fd-5b3a-461b-a913-2a4136166dff.png)